### ISSUE TITLE
Magento CE 2.0.0 to 2.0.1 upgrade

### ISSUE THREAD

**Issue by brankoajzele** [magento/magento2#3051]
I have an Ubuntu system with PHP 5.6.17, MySQL 5.6. cron job is set, as I can see cron_schedule table being filled nicely. 

I have installed my Magento 2.0.0 using composer create-project where it went through all the checks, etc., and it all went through nicely, few days ago. During which I could/can also access nicely admin and storefront areas, whereas my Magento is running in developer mode. 

I just tried doing an upgrade from Magento CE 2.0.0 to 2.0.1 and it failed, as shown in this video https://db.tt/oTV1mDIs.

Not sure what the issue is? Its not showing PHP version here, and its complaining that cron is not set, while it is.

Thoughts?

**Comment by xcomSteveJohnson** [comment]
I see this sometimes. Try this:

1. Close your browser window.
2. Manually delete from the file system the contents of Magento's `var/cache/*`, `var/page_cache/*`, `var/generation/*`, and `var/session/*

**Comment by mazhalai** [comment]
@ajzele can you confirm that you have the following crons setup: `/update/cron.php` and `/bin/magento setup:cron:run` Please see [this doc](http://devdocs.magento.com/guides/v2.0/comp-mgr/prereq/prereq_compman-updater.html) for more info

**Comment by brankoajzele** [comment]
Hey @mazhalai, good question. I can confirm I have both cron jobs set. Still no luck :(

**Comment by mazhalai** [comment]
If crons are running successfully you should have the following files:
`var/.update_cronjob_status` and `var/.setup_cronjob_status`. Can you please check contents of these files?

**Comment by brankoajzele** [comment]
Hey @mazhalai 

I can only see

`.update_cronjob_status` with content as follows:

{
    "readiness_checks": {
        "file_permissions_verified": true
    },
    "last_timestamp": 1453322944,
    "current_timestamp": 1453323005
}

Not seeing `var/.setup_cronjob_status`.

**Comment by brankoajzele** [comment]
Content of my var directory https://db.tt/u8kfkGbU

**Comment by mazhalai** [comment]
Then, there is some problem, [this](https://gist.github.com/mazhalai/ab290a1b96dea935c044) is what my `var/.setup_cronjob_status` looks like. One suggestion would be to output contents of setup cron into some temp file:

`*/1 * * * * /<full path to>php /<full path to>/bin/magento setup:cron:run >/tmp/setup.cron.log&`

and then check contents of `/tmp/setup.cron.log` in case an error occurred.

**Comment by brankoajzele** [comment]
Oh my... 

I had the following two entries:
`/bin/magento cron:run`
`/update/cron.php`

So you need overall of 3 cron jobs for Magento?
to get the cron_schedule table filling & executing
`/bin/magento cron:run`
to get the Component Manager and System Upgrade ticking
`/bin/magento setup:cron:run`
`/update/cron.php`

right?

The message is now much better https://db.tt/G0Haces8. However, IMHO, this message needs to be more clear in case of cron jobs, more detailed.

**Comment by mazhalai** [comment]
Exactly! ~~We are working on docs to clarify everything re cron jobs.~~ http://devdocs.magento.com/guides/v2.0/install-gde/install/post-install-config.html#post-install-cron

**Comment by brankoajzele** [comment]
Thank you @mazhalai 

I was able to get it finalised https://db.tt/1vGkUPjl

However I did notice one strange thing. During upgrade, it turned on maintenance, etc. However until it was able to finalise all, there was a timeframe of 2-3 minutes during which `git status` showed bunch of files as deleted as shown here https://db.tt/6GeghSTc, which resulted in storefront showing folder listing instead of maintenance mode https://db.tt/MNXEehwa.

To avoid confusion regarding git, I only added it all to locally git repo (git init, add / commit) to be able to track changes after upgrade. My Magento was originally installed with composer create-project, and not from Magento git.

Thoughts on the behaviour described above?
